src/bgw/job_stat.c

```diff
@@ -231,8 +235,13 @@ calculate_next_start_on_failure(TimestampTz finish_time, int consecutive_failure
 	}
 	PG_CATCH();
 	{
-		RollbackAndReleaseCurrentSubTransaction();
+		ErrorData *errdata = CopyErrorData();
+		ereport(LOG,
+				(errcode(ERRCODE_INTERNAL_ERROR),
+				 errmsg("calculating next start on failure "),
```

Comment:
@erimatnor This was feedback from the last PR about adding logging failures related to recalculation instead of ignoring them. 
This is just failure in recalculating start time and doesn't give a reason why the job failed.

Reply:
I understand why we want to print this error. I am just think we should make clear in the error what actually failed. Was it the original failure of the log or the failure to recompute. The message "calculating next start on failure" is ambiguous and doesn't explain what error occurred; it just gives the name of this function more or less (without the underscores).